aws/sdk/integration-tests/test.sh

```diff
@@ -16,7 +16,7 @@ for f in *; do
       else
          # The webassembly tests use a custom runner set in config.toml that
          # is not picked up when running the tests outside of the package
-         cd webassembly && cargo component test --all-features --all-targets && cd ..
+         cd webassembly && cargo component test --all-features --target wasm32-wasi && cd ..
```

Comment:
Shouldn't this still be all targets?

Reply:
I suspect "executing" `integration-tests/webassembly` is specifically for `wasm32-wasi`, but not `wasm32-unknown-unknown` that is mainly for browsers. Previously `all-targets` just included `wasm32-wasi` because that was the only target included in the cargo config for `integration-tests/webassembly`. But now that they are two of them, this line explicitly chooses `wasm32-wasi`. If that's true, maybe I should add a comment?

Reply:
Added explanatory comment in 8358943.